examples/light-switch-app/nrfconnect/Kconfig

```diff
@@ -40,6 +40,13 @@ config CHIP_ENABLE_SLEEPY_END_DEVICE_SUPPORT
 
 endif # NET_L2_OPENTHREAD
 
+if CHIP_WIFI
+
+config NRF_WIFI_LOW_POWER
```

Comment:
Why not enabling the low power mode for all our samples, i.e. by removing the NRF_WIFI_LOW_POWER=n from Kconfig.defaults? And what about NCS samples?

Reply:
> Why not enabling the low power mode for all our samples, i.e. by removing the NRF_WIFI_LOW_POWER=n from Kconfig.defaults?

Similarly to Thread, because we do not always need it. It only has sense for battery supplied devices. For devices like light bulb enabling low power mode and thus reducing device performance is not reasonable.

> And what about NCS samples?

NCS samples have this option enabled in their Kconfig files. It was introduced in the corresponding sdk-nrf PR: https://github.com/nrfconnect/sdk-nrf/pull/9978

Reply:
Ahh ok, I haven't noticed the ncs PR. Thanks.